.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         run: npm ci
 
       - name: Run ESLint
-        run: npx eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0
+        run: npx eslint . --report-unused-disable-directives --max-warnings 0
 
       - name: Run Prettier
         run: npx prettier . --check
```

Comment:
Had to update the workflow also as `--ext` is no longer usable like this, the file extensions are told in the config now.
Same is done in `package.json`